src/app/core/services/models/basic-rundown-entity.service.ts

```diff
@@ -17,4 +17,11 @@ export class BasicRundownEntityService {
       mode: RundownMode.INACTIVE,
     }
   }
+
+  public rehearsal(basicRundown: BasicRundown): BasicRundown {
```

Comment:
This a method name so we should use the verb `rehearse`

Reply:
Fixed